plugins/module_utils/helpers.py

```diff
@@ -27,3 +43,37 @@ def helper_cleanup_data(obj):
                          for k, v in obj.items() if k is not None and v is not None)
     else:
         return obj
+
+
+def helper_to_numeric_value(strs, value):
+    """Converts string values to integers
+
+    Parameters:
+        value: string value
+
+    Returns:
+        int: converted integer
+    """
```

Comment:
```suggestion
    """
    if value is None:
        return None
```

As per `zabbix_action` module example, which will prevent few of the bug occurrences once we use this function in that module as well.